### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by davegarrett**
This should be "chose" not "choosed", which isn't really an accepted word.

### CODE REVIEW

**Comment by davegarrett** [comment]
Added another commit here for a minor wording fix that snuck in with changing all extensions to be referred to by type ID name.

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -2781,7 +2781,7 @@ trial decrypt each record with the 1-RTT handshake keys
 until it finds one that decrypts properly, and then pick up
 the handshake from that point.
 
-If the server choosed to accept the "early_data" extension,
+If the server chose to accept the "early_data" extension,
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: I meant for this to be chooses

Comment (ekr) [inline draft-ietf-tls-tls13.md]: I agree that "choosed" isn't a word

**Comment by davegarrett** [comment]
Ok, I switched it to "chooses" instead & squashed my little fixes together here. There's also a fix for the "will MAY" bit that I think we discussed at some point but its fix didn't get in.

**Pull request merged**